release-notes/6.0/preview/api-diff/preview4/Asp.Net/6.0-preview4_Microsoft.AspNetCore.HttpLogging.md

```diff
@@ -0,0 +1,44 @@
+# Microsoft.AspNetCore.HttpLogging
+
+``` diff
++namespace Microsoft.AspNetCore.HttpLogging {
++    public enum HttpLoggingFields : long {
```

Comment:
FWIW the generated diffs here end up with the calculated values - I was confused as to why this wasn't done by bitwise-OR'ing fields together, and it turns out that it _is_ done this way, but you can only see it in the source [source.dot.net](https://source.dot.net/#Microsoft.AspNetCore.HttpLogging/HttpLoggingFields.cs,25a11615fdc00d78,references) .

It would be good to have the diffs generated, if possible, with these values to make it easier to understand

cc @tannergooding  - discussed with him on the C# Discord

Reply:
Notably the diff here is also missing the `[Flags]` attribute that is in the source/ref assembly

Reply:
i will take a look if we can fix this issue in the tooling.